cardinal_service_profile.post_update.php

```diff
@@ -125,3 +125,14 @@ function cardinal_service_profile_post_update_site_orgs() {
     }
   }
 }
+
+/**
+ * Removes stanford opportunity content type created by Stanford Profile.
+ * To prevent confusion between similarly named CT already configured.
+ */
+function cardinal_service_profile_post_update_remove_stanford_opportunities() {
```

Comment:
Why is this needed? the update hook won't actually do anything since the configuration is still there. Why not just change the name. Permissions should prevent the access to the content type anyways.

Reply:
@pookmish There are two Opportunity content types on the latest version of Cardinal Service: https://cardinalservice-dev.sites-pro.stanford.edu/admin/structure/types

Judging by the stanford profile, version 11.4.0 add an Opportunity content type to the profile: https://github.com/SU-SWS/stanford_profile/blob/11.x/CHANGELOG.md 

I could be wrong, but won't it come back since it's part of the Stanford Profile? Hence I added the post-update removal...

Reply:
Ooo. It does look like cardinal service profile is tracking it. Maybe I need to add it to the config ignore as well.

Reply:
That's why i suggest just changing the name of each content type. that way it won't break inheritance issues. Config ignore won't do anything. The site managers don't see it anyways except when filtering content on the /admin/content page.

Reply:
I did a masquerade and you're right they won't see. But won't renaming it just be overridden the next update?

Reply:
It will only cause merge conflicts if both the local line and the remote line change. So if we change the label in this repo AND stanford_profile changes the label, it will cause a merge conflict... which we just resolve like normal.

Reply:
I can see conflict merges being a pain but I don't have a better solution. So, I'll close this PR and get another one going.